content/sensu-go/5.0/api/assets.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Assets API"
+description: "Sensu Go assets API reference documentation"
+version: "5.0"
+product: "Sensu Go"
+menu:
+  sensu-go-5.0:
+    parent: api
+---
+
+## The `/assets` API endpoint
+
+### `/assets` (GET)
+
+The `/assets` API endpoint provides HTTP GET access to [asset][1] data.
+
+#### EXAMPLE {#assets-get-example}
+
+The following example demonstrates a request to the `/assets` API, resulting in
+a JSON Array containing [asset definitions][1].
+
+{{< highlight shell >}}
+curl -s http://127.0.0.1:8080/apis/core/v2/namespaces/default/assets -H "Authorization: Bearer TOKEN"
```

Comment:
We are going with `/api/` instead of `/apis/`, sorry about that!

Reply:
Fixed!